<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="NavListWidget" name="navList">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="infoSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="BookTableView" name="bookTable">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>24</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>292</width>
              <height>465</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="cover">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>160</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>No cover</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>6</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="author">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Айзек Азимов</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="title">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Роботы и империя</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>8</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="series">
               <property name="text">
                <string>&lt;b&gt;Серия:&lt;/b&gt; Роботы и империя (1)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tags">
               <property name="text">
                <string>&lt;b&gt;Теги:&lt;/b&gt; фантастика</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lang">
               <property name="text">
                <string>&lt;b&gt;Язык:&lt;/b&gt; ru</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="translator">
               <property name="text">
                <string>&lt;b&gt;Перевод:&lt;/b&gt; Иванов И. И.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSend_to">
     <property name="title">
      <string>Send to...</string>
     </property>
     <addaction name="actionConvertToDisk"/>
     <addaction name="actionSendToReader"/>
     <addaction name="actionSendBooksViaMail"/>
    </widget>
    <widget class="QMenu" name="menuCopyToCollection">
     <property name="title">
      <string>Copy to collection...</string>
     </property>
     <addaction name="actionAddInNewCollection"/>
    </widget>
    <addaction name="actionAddBooks"/>
    <addaction name="actionRemoveBooks"/>
    <addaction name="actionEditMetadata"/>
    <addaction name="separator"/>
    <addaction name="menuSend_to"/>
    <addaction name="separator"/>
    <addaction name="menuCopyToCollection"/>
    <addaction name="actionRemoveFromCollection"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSupportForum"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuCollection">
    <property name="title">
     <string>Collection</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewCollection"/>
     <addaction name="actionNewSmartCollection"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionEditCollection"/>
    <addaction name="actionDeleteCollection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCollection"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="ToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAddBooks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-add.png</normaloff>:/toolbar/tool-add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add files to Libro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionConvertToDisk">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-folder.png</normaloff>:/toolbar/tool-folder.png</iconset>
   </property>
   <property name="text">
    <string>Send to folder</string>
   </property>
   <property name="toolTip">
    <string>Send files to folder</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-settings.png</normaloff>:/toolbar/tool-settings.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Libro preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionRemoveBooks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-delete.png</normaloff>:/toolbar/tool-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove files from Libro</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionEditMetadata">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-edit.png</normaloff>:/toolbar/tool-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit metadata</string>
   </property>
   <property name="toolTip">
    <string>Edit files metadata</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSendToReader">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-reader.png</normaloff>:/toolbar/tool-reader.png</iconset>
   </property>
   <property name="text">
    <string>Send to reader device</string>
   </property>
   <property name="toolTip">
    <string>Send files to reader device</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSendBooksViaMail">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-mail.png</normaloff>:/toolbar/tool-mail.png</iconset>
   </property>
   <property name="text">
    <string>Send to email</string>
   </property>
   <property name="toolTip">
    <string>Send files to device email</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSupportForum">
   <property name="text">
    <string>Support forum</string>
   </property>
  </action>
  <action name="actionNewCollection">
   <property name="text">
    <string>Collection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewSmartCollection">
   <property name="text">
    <string>Smart collection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionEditCollection">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDeleteCollection">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCopyToNewCollection">
   <property name="text">
    <string>New collection</string>
   </property>
  </action>
  <action name="actionRemoveFromCollection">
   <property name="text">
    <string>Remove from collection</string>
   </property>
  </action>
  <action name="actionAddInNewCollection">
   <property name="text">
    <string>New collection</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookTableView</class>
   <extends>QTableView</extends>
   <header>.booktableview.h</header>
  </customwidget>
  <customwidget>
   <class>ToolBar</class>
   <extends>QToolBar</extends>
   <header>.toolbar.h</header>
  </customwidget>
  <customwidget>
   <class>NavListWidget</class>
   <extends>QListWidget</extends>
   <header>.navlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAddBooks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAddBooks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConvertToDisk</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSendToFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemoveBooks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionRemoveBooks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditMetadata</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionEditMetadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSendToReader</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSendToDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSendBooksViaMail</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSendViaMail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSupportForum</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSupportForum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionNewCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewSmartCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionNewSmartCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionEditCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionDeleteCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddInNewCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAddInNewCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemoveFromCollection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionRemoveFromCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionAddBooks()</slot>
  <slot>onActionSendToFolder()</slot>
  <slot>onActionSettings()</slot>
  <slot>onActionRemoveBooks()</slot>
  <slot>onActionAbout()</slot>
  <slot>onActionAboutQt()</slot>
  <slot>onActionEditMetadata()</slot>
  <slot>onActionViewLog()</slot>
  <slot>onActionSendToDevice()</slot>
  <slot>onActionSendViaMail()</slot>
  <slot>onActionHelp()</slot>
  <slot>onActionSupportForum()</slot>
  <slot>onActionNewCollection()</slot>
  <slot>onActionNewSmartCollection()</slot>
  <slot>onActionEditCollection()</slot>
  <slot>onActionDeleteCollection()</slot>
  <slot>onActionAddInNewCollection()</slot>
  <slot>onActionRemoveFromCollection()</slot>
 </slots>
</ui>
